network_test.go

```diff
@@ -312,63 +313,65 @@ func testNetworkMachineCNI(t *testing.T, useConfFile bool) {
 		require.NoError(t, err, "cni conf should parse")
 	}
 
-	numVMs := 10
-	vmIPs := make(chan string, numVMs)
+	if runtime.GOARCH != "arm64" {
```

Comment:
- Let's use t.Skip instead
- If possible, we should skip this test entirely. It doesn't make much sense to setup the VM, if we don't run any validations against the VM.

Reply:
sure, let me try in that case.

Reply:
After using t.skip() here, **TestMicroVMExecution** test got failed in the BuildKite because we are skipping and not allowing the VM's to perform the test.
https://buildkite.com/firecracker-microvm/vvejella-firecracker-go-sdk-arm/builds/27#01852ca2-9f4c-41aa-b2a4-0d6e834c6399:~:text=FAIL%3A-,TestMicroVMExecution,-(30.03s)

Reply:
I don't think they are related. The VM here and TestMicroVMExecution's VM are different.

Reply:
May be it is causing flaky test and now I test is fine